test/integration/pkcs-sign-verify.int.c

```diff
@@ -700,7 +699,6 @@ RSA *template_to_rsa_pub_key(CK_ATTRIBUTE_PTR attrs, CK_ULONG attr_len) {
 
     return ssl_rsa_key;
 
-error:
     BN_free(n);
     BN_free(e);
     RSA_free(ssl_rsa_key);
```

Comment:
Close you needed to delete the stuff under the label too... Ill take care of this.